src/test/c/CMakeLists.txt

```
@@ -2,28 +2,23 @@ add_library(Catch INTERFACE)
 target_include_directories(Catch INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/catch")
 
 add_seasocks_executable(AllTests
-    test_main.cpp
-    ConnectionTests.cpp
-    CrackedUriTests.cpp
-    HeaderMapTests.cpp
-    HtmlTests.cpp
-    HybiTests.cpp
-    JsonTests.cpp
-    MockServerImpl.h
-    ServerTests.cpp
-    ToStringTests.cpp
-    EmbeddedContentTests.cpp
-    ResponseBuilderTests.cpp
-    ResponseTests.cpp
-    StringUtilTests.cpp
-    RequestTest.cpp
-)
+        test_main.cpp
+        ConnectionTests.cpp
+        CrackedUriTests.cpp
+        HeaderMapTests.cpp
+        HtmlTests.cpp
+        HybiTests.cpp
+        JsonTests.cpp
+        MockServerImpl.h
+        ServerTests.cpp
+        ToStringTests.cpp
+        EmbeddedContentTests.cpp
+        ResponseBuilderTests.cpp
+        ResponseTests.cpp
+        StringUtilTests.cpp
+        RequestTest.cpp
+        )
 
-seasocks_link_libraries(AllTests PRIVATE Catch)
+target_link_libraries(AllTests PRIVATE Catch)
```

Otherwise, a shared build fails.
```suggestion
target_link_libraries(AllTests PRIVATE Catch Threads::Threads)
```

---

@mattgodbolt Chech this one too.